### DISCUSSION TITLE
死活都连不上

### DISCUSSION THREAD

**Discussion by pig3g** [zzzgydi/clash-verge#880] [Q&A]
不知道哪里出了问题，订阅后，开启代理开关，选节点，但是连不上，后来卸载重装，猜测是端口的问题也改过了问题依旧
其他软件都没问题，就clash verge这样
![QQ截图20231102204130](https://github.com/zzzgydi/clash-verge/assets/61103443/0b575338-e412-47ae-bea8-61ebdc392394)
![QQ截图20231102204142](https://github.com/zzzgydi/clash-verge/assets/61103443/23d3eba4-1380-474c-96bb-3a7f579741f0)
![QQ截图20231102204147](https://github.com/zzzgydi/clash-verge/assets/61103443/6221501e-a8db-4a5f-96c0-81ab8b411f39)

**Comment by sakukaze-rin** [comment]
设置里面，找到“Clash内核”选项，点击进入，选择“Clash Meta”，然后在“Clash内核”上面有个“Clash字段”，点击进入后把有黄色感叹号的选项都进行勾选。这样应该可以解决问题。
